sn_interface/src/messaging/system/mod.rs

```diff
@@ -57,7 +57,7 @@ pub enum AntiEntropyKind {
 #[derive(Clone, PartialEq, Serialize, Deserialize, custom_debug::Debug)]
 #[allow(clippy::large_enum_variant, clippy::derive_partial_eq_without_eq)]
 /// Message sent over the among nodes
-pub enum SystemMsg {
+pub enum Node2NodeMsg {
```

Comment:
I'd say it's clear enough as `NodeMsg`.

```suggestion
pub enum NodeMsg {
```